### PR TITLE
Upgrade to Mutiny Zero 0.4.2

### PR DESCRIPTION
**Pull request by jponge**

### CODE REVIEW

**Comment by jponge** [comment]
Hum not sure what had already been merged before, but this brings up to the latest version 😉

File: smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/ce/KafkaSourceBatchWithCloudEventsTest.java [hunk 1]
```diff
@@ -39,6 +34,11 @@
 import io.smallrye.reactive.messaging.kafka.KafkaConnectorIncomingConfiguration;
 import io.smallrye.reactive.messaging.kafka.KafkaRecord;
 import io.smallrye.reactive.messaging.kafka.KafkaRecordBatch;
+import io.smallrye.reactive.messaging.kafka.base.BufferSerde;
```
Comment (cescoffier) [inline smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/ce/KafkaSourceBatchWithCloudEventsTest.java:37]: @ozangunalp yet another import ordering issue?

File: smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/ce/KafkaSourceWithCloudEventsTest.java [hunk 2]
```diff
@@ -49,8 +49,7 @@ public class KafkaSourceWithCloudEventsTest extends KafkaCompanionTestBase {
     public static void setup() {
         companion.registerSerde(JsonObject.class, Serdes.serdeFrom(
                 new JsonObjectSerde.JsonObjectSerializer(),
-                new JsonObjectSerde.JsonObjectDeserializer())
-        );
+                new JsonObjectSerde.JsonObjectDeserializer()));
```
Comment (cescoffier) [inline smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/ce/KafkaSourceWithCloudEventsTest.java:52]: Looks like we also have a formatter issue.

**Review by cescoffier — CHANGES_REQUESTED** [review]
Is it just a bump of the TCK? 

I'm more worried about the import and format changes.

**Comment by jponge** [comment]
So:

1. it's a Mutiny Zero bump,
2. formatting is what I got locally after a proper Maven build, I guess it's what should be under VCS

**Comment by jponge** [comment]
Yes but there's a version bump.

**Comment by ozangunalp** [comment]
When I rebase the branch onto `upstream/main` I get "dropping 2fdd947915bc48d6f39b8b1bfbf6fd14f31d37c1 Use Mutiny Zero 0.4.2 -- patch contents already upstream"

**Comment by jponge** [comment]
Ok stupid me then, I had already opened that PR before 🤣

**Comment by ozangunalp** [comment]
Back to the future

**Pull request closed without merging**